src/isSet.js

```diff
@@ -0,0 +1,25 @@
+import { is } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+/**
+ * Predicate for determining if a provided value is an instance of a Set.
+ *
+ * @func isSet
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isSet}
```

Comment:
`@see {@link RA.isMap|isMap}` - it doesn't make sense to link to itself. Somebody else is working on `isMap` PR, so let's link to it.

Reply:
I think I have done so now